elections/steering/2022/README.md

```diff
@@ -131,20 +132,20 @@ Examples of contributions that would NOT be considered:
   public announcement is a weekend.
 -->
 
-| Date                    | Event                                                                 |
-|:------------------------|:----------------------------------------------------------------------|
-| Friday, July 22         | Steering Committee selects Election Committee                         |
-| Monday, August 15       | Announcement of Election and publication of voters.md                 |
-| ???, August ??          | Steering Committee Q+A for the candidates                             |
-| Tuesday, August 30      | Candidate nominations due at the end of the day in AoE time           |
-| Friday, September 2     | All candidate bios due at the end of the day in AoE time              |
-| Tuesday, September 6    | Election Begins via email ballots                                     |
-| Friday, September 16    | Deadline to submit voter exception requests                           |
-| Thursday, September 29  | Election Closes at the end of the day in AoE time                     |
-| Friday, September 30    | Private announcement of Results to SC members not up for election     |
-| Sunday, October 2       | Private announcement of Results to all candidates                     |
-| Monday, October 3       | Public announcement of Results at Public Steering Committee Meeting   |
-| Tuesday, October 4 ???  | Election Retro                                                        |
+| Date                   | Event                                               |
+|:-----------------------|:----------------------------------------------------|
+| Friday, July 22        | Steering Committee selects Election Committee       |
+| Monday, August 15      | Announcement of Election and publication of voters.md |
+| ???, August ??         | Steering Committee Q+A for the candidates           |
+| Tuesday, August 30     | Candidate nominations due at the end of the day in AoE time |
+| Friday, September 2    | All candidate bios due at the end of the day in AoE time |
+| Tuesday, September 6   | Election Begins via email ballots                   |
+| Friday, September 16   | Deadline to submit voter exception requests         |
+| Thursday, September 29 | Election Closes at the end of the day in AoE time   |
+| Friday, September 30   | Private announcement of Results to SC members not up for election |
+| Sunday, October 2      | Private announcement of Results to all candidates   |
+| Monday, October 3      | Public announcement of Results at Public Steering Committee Meeting |
+| Tuesday, October 4 ??? | Election Retro                                      |
```

Comment:
Auto-reformat?

Reply:
Dammit, yes.  That's a widget that helps me create MD tables, I forgot to turn it off.  Unless you like the reformatting?